src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -37,10 +37,29 @@ public static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Pref
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
 
+    /**
+     * Throws ParseException if {@code entity} provided is not a valid entity in the address book.
+     * Leading and trailing whitespaces will be trimmed.
+     */
+    public static void requireValidEntity(String entity) throws ParseException {
+        requireNonNull(entity);
+        String trimmedEntity = entity.trim();
+
+        // TODO: Not sure if this is magic string?
+        switch (trimmedEntity) {
+        case "contact", "company", "job", "all":
```

Comment:
If you don't wanna have the magic strings, we can just store the entity words into Person, Job, and Company as constants.

Reply:
I guess that is fine, but does that mean we should change the entity word of all commands as well?

Also if we are going by this, I am not exactly sure where to store "all".

Reply:
hmm yeah not sure where to store that lol